openedx/core/djangoapps/credentials/signals.py

```diff
@@ -0,0 +1,88 @@
+"""
+This file contains signal handlers for credentials-related functionality.
+"""
+from logging import getLogger
+
+from course_modes.models import CourseMode
+from django.contrib.sites.models import Site
+from lms.djangoapps.certificates.models import CertificateStatuses, GeneratedCertificate
+from lms.djangoapps.grades.course_grade_factory import CourseGradeFactory
+from openedx.core.djangoapps.catalog.utils import get_programs
+
+from .tasks.v1.tasks import send_grade_to_credentials
+
+log = getLogger(__name__)
+
+
+INTERESTING_MODES = CourseMode.VERIFIED_MODES + CourseMode.CREDIT_MODES
```

Comment:
nit: I don't know how I feel about interesting but I also don't have a better name

Reply:
Added a comment here, as we talked about offline. Hopefully helps anyone looking at it later.